cmd/cqld/main.go

```diff
@@ -144,3 +153,51 @@ func main() {
 
 	log.Info("server stopped")
 }
+
+func registerNodeToBP(timeout time.Duration) (err error) {
```

Comment:
why you copy this function from "cql-minerd"
if it's common usage why not to put it into some package